### ISSUE TITLE
Success & Danger Panel CTA buttons fail AA contrast

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1166]
Success & Danger Panel CTA buttons fail AA contrast. See lexicon examples. Looks to be caused by immutable button colour styles.

Also, lexicon timeline panel has a contrast issue on the timeline avatar and view history naked button. Also 2 date/time related validation errors.
